docroot/themes/contrib/civictheme/includes/slider.inc

```diff
@@ -87,3 +82,15 @@ function civictheme_preprocess_paragraph__civictheme_slider_slide(&$variables) {
     ];
   }
 }
+
+/**
+ * Implements template_preprocess_paragraph().
+ */
+function civictheme_preprocess_paragraph__civictheme_slider_slide_ref(&$variables) {
+  /** @var \Drupal\paragraphs\Entity\Paragraph $paragraph */
+  $paragraph = $variables['paragraph'];
+  $parent = $paragraph->_referringItem->getEntity();
+  // @todo add theme field to reference slide.
+  $theme = civictheme_get_theme_value($parent);
+  $paragraph->civic_theme = $theme;
```

Comment:
please remove this - we do not want any "pseudo" fields created ad-hoc on standard Drupal entities.